File: rest/recording/list-get-example-4/list-get-example-4.4.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
client = Twilio::REST::Client.new(account_sid, auth_token)

# Loop over recordings and print out a property for each one
recordings = client.account.recordings.list("date_created<": '2016-10-15',
                                            "date_created>": '2016-10-12')
recordings.each do |recording|
  puts recording.duration
end

File: rest/recording/list-get-example-4/list-get-example-4.4.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string AuthToken = "your_auth_token";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    var recordings = twilio.ListRecordings(
        null,
        dateCreatedLessThanOrEqual: new DateTime(2016, 10, 15),
        dateCreatedGreaterThanOrEqual: new DateTime(2016, 10, 12)
    );

    foreach (var recording in recordings.Recordings)
    {
      Console.WriteLine(recording.Duration);
    }
  }
}
